src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -119,6 +119,9 @@ protected function getDumpHeader()
     addEventListener = function (e, n, cb) {
         e.addEventListener(n, cb, false);
     };
+    
+var isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```

Comment:
isMac intermediary variable can be removed by merging this with the ternary check on the line below.
I'd also merge this new `var` declaration with the one above, eg. aftter `idRx`